config/database.yml

```diff
@@ -3,21 +3,21 @@ default: &default
   pool: 5
   timeout: 5000
   host: localhost
-  port: <%= ENV['DB_PORT'] || '5432' %>
-  username: <%= ENV['DB_USER'] || ENV['USER'] || 'postgres' %>
```

Comment:
We need these configured because the different dbs will be configured with different creds